src/components/DataTable/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Table } from 'react-bootstrap';
+
+import style from './index.module.scss';
+
```

Comment:
please put a comment to briefly explain how to use this component and what kind of data are to be passed as props